Sources/MetricsTestKit/Docs.docc/index.md

```diff
@@ -9,29 +9,24 @@ This module offers a ``TestMetrics`` type which you use to bootstrap the metrics
 ### Example
 
 ```swift
-import XCTest
 import Metrics
 import MetricsTestKit
+import Testing
 
-final class ExampleTests: XCTestCase {
-    var metrics: TestMetrics! = TestMetrics()
+struct ExampleTests {
```

Comment:
I wonder if we should show both examples, one with explicitly using a MetricsFactory and passing it around, and a second one with bootstrap (the existing one).

Especially for testing reasons, I think it's good to nudge adopters into explicitly passing around MetricsFactory, especially to components that can be unit-tested.